### ISSUE TITLE
[STYLE]: Re-Designing the homePage

### ISSUE THREAD

**Issue by senthil-k8s** [jayk-gupta/web-resources-project#500]
### Is your feature request related to a problem? Please describe
The current homepage does not contain any valuable points or meaningful content. The Landing page should describe what the current website is meant for and its functionalities. But unfortunately, we don't have any feature at all.

### Describe the solution you'd like

1. User-Friendly Design: The landing page should have an intuitive and visually appealing design to engage users and encourage them to explore the available resources.
2. Resource Categories: Organize the resources into distinct categories, such as HTML, CSS, JavaScript, frameworks, and libraries, allowing users to quickly find relevant learning materials.
3. Search Functionality: Implement a search feature that enables users to search for specific topics, tutorials, or keywords within the available resources.
4. Featured Content: Highlight key resources, tutorials, or projects on the landing page to provide immediate value and showcase the breadth of available learning materials.
5. Responsive Design: Ensure that the landing page is fully responsive, providing an optimal user experience across different devices and screen sizes.
6. Social Sharing: Include social sharing buttons to allow users to easily share the resources they find useful with their network, promoting the platform and its content.

### Additional context

I would like to work under GSSoC'23
Please assign me this issue

**Comment by jayk-gupta** [comment]
working on it?

**Comment by senthil-k8s** [comment]
yeah, am working on it. I have raised PR for adding resources. so, just now checked notification. Will raise PR by this weekend.
